domains/noah.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "thatsateddy",
+        "email": "noah@gmx.se"
+    },
+    "record": {
+        "A": ["207.127.90.134"]
```

Comment:
I cannot reach your website, keeps on loading

Reply:
> I cannot reach your website, keeps on loading

It runs on various ports